src/routers/features/cgn/index.ts

```diff
@@ -91,6 +98,22 @@ addHandler(cgnRouter, "get", addPrefix("/activation"), (_, res) =>
         instance_id: { id },
         status: StatusEnum.COMPLETED
       };
+
+      const currentPreference = servicesPreferences.get(
+        cgnServiceId as ServiceId
+      );
+
+      const increasedSettingsVersion = currentPreference
+        ? (((currentPreference.settings_version as number) +
+            1) as ServicePreference["settings_version"])
+        : (0 as ServicePreference["settings_version"]);
+      servicesPreferences.set(cgnServiceId as ServiceId, {
+        is_inbox_enabled: true,
+        is_email_enabled: faker.datatype.boolean(),
```

Comment:
```suggestion
        is_email_enabled:currentPreference?.is_email_enabled ??
          getRandomValue(false, faker.datatype.boolean(), "services"),
```
the same of the other part: we should avoid the dev madness with a value that could change on every API invocation